myokit/formats/sbml/__init__.py

```diff
@@ -7,12 +7,14 @@
 from __future__ import absolute_import, division
 from __future__ import print_function, unicode_literals
 
-from ._importer import SBMLImporter, SBMLError  # noqa
+from ._importer import SBMLImporter  # noqa
+from ._parser import SBMLError, SBMLParser  # noqa
 
 
 # Importers
 _importers = {
```

Comment:
I'd write this so that:

1. The SBMLImporter implements the Importer interface, and is the only class listed here
2. The SBMLImporter uses the SBMLParser, which doesn't implement any specific interface